### DISCUSSION TITLE
Create local instance with dummy data

### DISCUSSION THREAD

**Discussion by rettetdemdativ** [mastodon/mastodon#28007] [Q&A]
I'm trying to create a local instance for testing purposes and fill it with dummy data (fake posts, fake users, etc.). What's the best way to approach this? Do I use the API, do I manipulate the DB directly? Is it possible to create trends from this? Can anyone point me in the right direction please?

**Comment by mjankowski** [comment]
I realize this is several years old, but:

- There is a `dev:populate_sample_data` rake task that does some of this
- You can just use rails console and fabricator definitions to make whatever you want, and/or just interact with locally running dev mode UI to make stuff
